app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -31,7 +31,12 @@ def shared_update
 
       idv_session.verify_info_step_document_capture_session_uuid = document_capture_session.uuid
 
-      Idv::Agent.new(pii).proof_resolution(
+
+
+      user_pii = pii
+      user_pii[:best_effort_phone_number_for_socure] = best_effort_phone
```

Comment:
Because we're doing something slightly unusual, we tack this onto the PII bundle so it's available for Socure, but omit it other places it might be used.

The naming is meant to convey that we accept that the phone number may not be present and shouldn't be relied on for other purposes.